### PR TITLE
Added new shop form page, route, controller test

### PR DESCRIPTION
**Pull request by katief906**
Created a NewShopForm container
Create a :create route in routes.rb
Created a :create method in API::V1::ShopsController along with shop_params
Create a controller test

### CODE REVIEW

**Comment by kwong402** [comment]
Katie addressed merge conflicts

File: app/javascript/react/components/NewShopForm.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+          headers: {
+            "Content-Type": "application/json"
+          },
+          body: JSON.stringify({ shop: shopRecord })
+        })
+        if (!response.ok) {
+          const errorMessage = `${response.status} (${response.statusText})`
+          throw new Error(errorMessage)
+        }
+        console.log("Shop was added successfully!")
```
Comment (dovakeen118) [inline app/javascript/react/components/NewShopForm.js]: we can remove console log statements like this

File: app/javascript/react/components/NewShopForm.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        />
+      </label>
+
+      <label>
+        Shop Picture (URL)
+        <input 
+          type="text"
+          name="picture"
+          value={shopRecord.picture}
+          onChange={handleInputChange}
```
Comment (dovakeen118) [inline app/javascript/react/components/NewShopForm.js]: image uploading on the react side should be handled with a drop zone and have an S3 bucket set up - let's remove this image property for now, and in a separate feature work on image uploads

File: app/javascript/react/components/ShopIndexPage.js [hunk 3]
```diff
@@ -36,8 +37,16 @@ const ShopIndexPage = () => {
...
   return (
     <div>
-    <h1>All Bubble Tea Shops</h1>
-      {shopTiles}
+      <div>
+        <h1>All Bubble Tea Shops</h1>
+        {shopTiles}
+      </div>
+
+      <Link to={`/shops/new`}>
```
Comment (dovakeen118) [inline app/javascript/react/components/ShopIndexPage.js]: this path is not dynamic so it doesn't need to be wrapped in curly brackets with back ticks

**Review by dovakeen118 — COMMENTED** [review]
looking good! let's remove the image property in the form for now until you are ready to fully implement the image upload feature

**Comment by katief906** [comment]
@dovakeen118 just addressed all of your comments from the last commit

**Pull request merged**